plugin/core/publish.gradle

```diff
@@ -1,4 +1,4 @@
-version = '0.9.2'
+version = '0.9.3'
```

Comment:
I believe we can take this change out of this PR. We will increase the version as part of the release PR.

Reply:
ok, I'll undo it